### PR TITLE
added contribute to crowd map and pods update

### PR DESCRIPTION
**Pull request by jankrzempek**
What was done:

Ability to decide if You would like to contribute to crowd Map or not
Updated pod file

https://trello.com/c/XkbtqPbo

### CODE REVIEW

File: AirCasting/CreateSessionViews/ChooseSessionTypeView.swift [hunk 1]
```diff
@@ -101,7 +102,7 @@ struct ChooseSessionTypeView: View {
             _ = bluetoothManager.centralManager
             didTapFixedSession = true
         } else if CBCentralManager.authorization == .allowedAlways,
-           bluetoothManager.centralManager.state == .poweredOn {
+                  bluetoothManager.centralManager.state == .poweredOn {
```
Comment (pawelgil) [inline AirCasting/CreateSessionViews/ChooseSessionTypeView.swift:105]: make sure it's not over-indented plz :)

File: Podfile.lock [hunk 2]
```diff
@@ -1,83 +1,101 @@
 PODS:
-  - Firebase/Analytics (7.5.0):
+  - Firebase/Analytics (8.3.0):
```
Comment (pawelgil) [inline Podfile.lock:2]: I don't see a connection between task and updates, if those are not needed to complete the task please make a separate PR

File: AirCasting/CreateSessionViews/ChooseSessionTypeView.swift (hunk 1 above)
Comment (jankrzempek) [reply AirCasting/CreateSessionViews/ChooseSessionTypeView.swift:105]: After clicking cmd+a and then ctr+I it goes to that state. I don't know what to think about it

Comment (pawelgil) [reply AirCasting/CreateSessionViews/ChooseSessionTypeView.swift:105]: Personally I use ctrl+l only for the code I'm actually changing in a given PR. Doing cmd+a + ctrl+l can lead to unneeded conflicts and stuff like that. It's better to have a full control :)

**Review by pawelgil — APPROVED** [review]

File: AirCasting/AuthViews/LifeTimeEventsProvider.swift [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
  import Foundation
 
- class UserDefaultProtocol: ObservableObject {
+ class LifeTimeEventsProvider: ObservableObject {
```
Comment (smialko) [inline AirCasting/AuthViews/LifeTimeEventsProvider.swift:6]: Cała klasa jest przesunięta w prawo, popraw proszę wcięcie ;)

**Review by smialko — APPROVED** [review]
🍉

File: AirCasting/AuthViews/LifeTimeEventsProvider.swift (hunk 3 above)
Comment (smialko) [reply AirCasting/AuthViews/LifeTimeEventsProvider.swift:6]: Ups, to już jest zmregowane ;D
